artemis-server/src/main/java/org/apache/activemq/artemis/core/management/impl/ClusterConnectionControlImpl.java

```diff
@@ -321,5 +321,18 @@ public Map<String, Object> getBridgeMetrics(String nodeId) {
 
    }
 
+   @Override
+   public long getProducerWindowSize() {
+      if (AuditLogger.isBaseLoggingEnabled()) {
+         AuditLogger.getTopology(this.clusterConnection);
```

Comment:
This seems wrong, surely getProducerWindowSize() shouldn't be audit logging about getting topology?

Reply:
Nice catch. I skimmed right over that. I fixed it with 42ddf99ca8846218ee7b49d97a4f5087318c15e3.